Majasdarbs/10-Galadarbs/public/script.js

```diff
@@ -0,0 +1,48 @@
+//const userInput = document.getElementById("cityinput").value;
+//const userInput = document.querySelector("#cityinput");
+
+const userInput = "Riga";
+const KELVIN = 273;
+//const key = process.env.API_KEY;
+const key = "3045dd712ffe6e702e3245525ac7fa38";
```

Comment:
better off moving this to a separate config.js file

But then in order to import/export modules, you need to use modules type of the js (by default it is set to commonjs).